superset-embedded-sdk/.gitignore

```diff
@@ -0,0 +1,3 @@
+bundle
+dist
+lib
```

Comment:
Any need to add `node_modules` or other typical cruft here?

Reply:
nah, the root .gitignore file also applies here